roles/dropwizard-api/files/start-container.j2

```diff
@@ -35,15 +35,17 @@ fi
 uid="$(id -u {{ user }})"
 gid="$(id -g {{ user }})"
 
-# Stop and destroy container if it's already running.
-echo "Stop existing container"
-docker rm -f $api &
+# Stop and destroy container if it's already running
+if [[ "$(docker ps -aq -f name=$api)" ]]; then
+    echo "Stop existing container"
+    docker rm -f "$api" &
```

Comment:
I don't think we should run this command in the background anymore, since we want the program to fail if docker rm fails at this point.

Reply:
good catch, I forgot to change this back. Will change.